docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -225,8 +223,7 @@ services:
     networks:
       - gitea
     volumes:
--     - ./gitea:/data
-+     - gitea:/data
+      - ./gitea:/data
```

Comment:
If there is volume defined than this should be `- gitea:/data`

Reply:
Done.